QATCH/processors/Analyze.py

```
@@ -9358,6 +9353,11 @@ def reject_outliers(data, m):
             plotWidget.setLayout(mp_layout)
             self.parent.results_split.replaceWidget(1, plotWidget)
 
+            # force layout redraw now, and on any resize event
+            sc.draw_idle()
+            filter = ResizeFilter(self)
```

Ruff (0.15.12)

[error] Variable `filter` is shadowing a Python builtin

(A001)